data/brands/shop/books.json

```diff
@@ -326,6 +326,17 @@
         "shop": "books"
       }
     },
+    {
+      "displayName": "Foyles",
+      "id": "foyles-4cb062",
+      "locationSet": {"include": ["gb"]},
```

Comment:
The shops are only in England so could this be changed to:

```json
"locationSet": {"include": ["gb-eng"]},
```

Reply:
Done, thanks.